newIDE/app/src/AssetStore/ExampleStore/index.js

```diff
@@ -39,13 +43,33 @@ export const ExampleStore = ({ isOpening, onOpen }: Props) => {
     setSearchText,
   } = React.useContext(ExampleStoreContext);
 
+  const shouldAutofocusSearchbar = useShouldAutofocusSearchbar();
+  const searchBarRef = React.useRef<?SearchBarInterface>(null);
+
   React.useEffect(
     () => {
       fetchExamplesAndFilters();
     },
     [fetchExamplesAndFilters]
   );
 
+  React.useEffect(
+    () => {
+      if (focusOnMount && shouldAutofocusSearchbar && searchBarRef.current)
+        searchBarRef.current.focus();
+    },
+    [shouldAutofocusSearchbar, focusOnMount]
+  );
+
+  const tagsHandler = React.useMemo(
+    () => ({
+      add: filtersState.addFilter,
+      remove: filtersState.removeFilter,
+      chosenTags: Array.from(filtersState.chosenFilters),
+    }),
+    [filtersState]
```

Comment:
A bit nervous about this useMemo, but I guess if addFilter/removeFilter/chosenFilters change, then filtersState change so it should be correct.